unikraft/arch/architecture.go

```diff
@@ -15,6 +15,63 @@ import (
 	"kraftkit.sh/unikraft/component"
 )
 
+type ArchitectureName string
+
+const (
+	ArchitectureUnknown = ArchitectureName("unknown")
+	ArchitectureX86_64  = ArchitectureName("x86_64")
+	ArchitectureArm64   = ArchitectureName("arm64")
+	ArchitectureArm     = ArchitectureName("arm")
+)
+
+// String implements fmt.Stringer
+func (ht ArchitectureName) String() string {
+	return string(ht)
+}
+
+// ArchitectureByName returns the architecture for a given name.
+// If the name is not known, it returns it unchanged.
+func ArchitectureByName(name string) ArchitectureName {
+	architectures := ArchitecturesByName()
+	if _, ok := architectures[name]; !ok {
+		return ArchitectureName(name)
```

Comment:
Shouldn't this be `unknown`?

Reply:
That's what I thought when I copied it from the Platform code

Reply:
Ok. Platform code should later be updated :)